src/OSPSuite.Presentation/Presenters/Charts/ChartEditorPresenter.cs

```diff
@@ -649,6 +665,11 @@ public void Refresh()
          updateUsedColumns();
       }
 
+      public void SetGroupRowNamingReplacements(Dictionary<string, string> groupingNamesReplacementDictionary)
```

Comment:
THe way you defined this is not consistent. Here it's a function, in the databrowser, it's a property.
Why?